lms/static/js/inline_analytics.js

```diff
@@ -378,6 +371,23 @@ window.InlineAnalytics = (function() {
     }
 
 
+    function convertNameToArray(choiceNameString) {
+        // Convert the choice name string to a list
+        choiceNameString = choiceNameString.substring(1, choiceNameString.length -1).replace(/'/g, "");
+        return choiceNameString.split(", ");
+    }
+
+    
```

Comment:
it's too bad that this was necessary.  maybe consider making the top level data structure an "object" / "dict"  with part_ids as the key instead?